addons/crm/static/src/js/tours/crm.js

```diff
@@ -35,9 +35,10 @@ tour.register('crm_tour', {
     trigger: ".o_kanban_quick_create .o_field_widget[name='name']",
```

Comment:
- Rewrite 'This change allows certain widgets as the FieldMonetary (which keeps the
same input while detaching/reattaching it to the DOM) to keep consistent
handlers.'
-Make clear the link with keynav